### PR TITLE
2.x - Update tests and formatting.

### PR DESCRIPTION
**Pull request by markstory**
* Upgrade to PHPUnit 8.0
* Apply latest codesniffer rules.

### CODE REVIEW

**Comment by ADmad** [comment]
I took the liberty of updating typehints too.

File: src/AuthenticationService.php [hunk 1]
```diff
@@ -151,17 +155,17 @@ public function loadAuthenticator($name, array $config = [])
...
      */
-    public function loadIdentifier($name, array $config = [])
+    public function loadIdentifier(string $name, array $config = []): IdentifierInterface
     {
         return $this->identifiers()->load($name, $config);
     }
 
     /**
-     * {@inheritDoc}
+     * @inheritdoc
```
Comment (dereuromark) [inline src/AuthenticationService.php]: wrong one, thats not the partial overwrite

Comment (ADmad) [reply src/AuthenticationService.php]: Casualty of search - replace :)

**Comment by dereuromark** [comment]
I fixed those also in my branch :)

**Pull request merged**